### PR TITLE
Update to a more modern SVG sprite compiler

### PR DESCRIPTION
**Pull request by thisisdano**
Updating to `gulp-svgstore`. Notes in the comments

### CODE REVIEW

**Comment by mgwalker** [comment]
Tested on an ARM64 arch `node:16` docker container and the install worked, which addresses the primary concern in #25. I don't have a test project handy to try executing it, but I'll see if I can put something together.

**Review by mejiaj — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ npm-debug.log*
 test/
 path/
 *.tgz
+assets/
+sass/
```
Comment (thisisdano) [inline .gitignore:8]: These are generated when testing, but should be ignored

File: gulpfile.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const replace = require("gulp-replace");
 const sass = require("gulp-sass")(require("sass-embedded"));
 const sourcemaps = require("gulp-sourcemaps");
 const del = require("del");
-const svgSprite = require("gulp-svg-sprite");
```
Comment (thisisdano) [inline gulpfile.js:9]: Use the new sprite builder

File: gulpfile.js [hunk 3]
```diff
@@ -182,44 +182,24 @@ function watchSass() {
...
-      spacing: {
-        // Add padding
-        padding: 0,
-      },
-    },
-    mode: {
-      symbol: true, // Activate the «symbol» mode
-    },
-  };
-
```
Comment (thisisdano) [inline gulpfile.js:204]: This config does not apply to the new package. As far as I can tell, this is not necessary as long as we continue to use 24x24 SVGs with `height`, `width`, and `viewBox` set properly.

File: gulpfile.js [hunk 4]
```diff
@@ -182,44 +182,24 @@ function watchSass() {
...
   })
-    .pipe(svgSprite(config))
+    .pipe(svgSprite())
     .on("error", handleError)
     .pipe(dest(`${paths.dist.img}`));
 }
 
 function renameSprite() {
-  return src(`${paths.dist.img}/symbol/svg/sprite.symbol.svg`.replaceAll("//", "/"), {
+  return src(`${paths.dist.img}/usa-icons.svg`.replaceAll("//", "/"), {
```
Comment (thisisdano) [inline gulpfile.js:194]: Point to the new outputted file

File: package.json [hunk 5]
```diff
@@ -24,17 +24,21 @@
...
-    "gulp-postcss": "^9.0.1",
-    "gulp-rename": "^2.0.0",
-    "gulp-replace": "^1.1.3",
-    "gulp-sass": "^5.0.0",
-    "gulp-sourcemaps": "^3.0.0",
-    "gulp-svg-sprite": "^1.3.7",
-    "postcss": "^8.4.8",
-    "postcss-csso": "^5.0.1",
-    "sass-embedded": "^1.49.9"
+    "autoprefixer": "10.4.7",
```
Comment (thisisdano) [inline package.json:27]: Update `autoprefixer`

File: package.json [hunk 6]
```diff
@@ -24,17 +24,21 @@
...
+    "del": "6.0.0",
+    "gulp": "4.0.2",
+    "gulp-postcss": "9.0.1",
+    "gulp-rename": "2.0.0",
+    "gulp-replace": "1.1.3",
+    "gulp-sass": "5.1.0",
```
Comment (thisisdano) [inline package.json:33]: Update `gulp-sass`

File: package.json [hunk 7]
```diff
@@ -24,17 +24,21 @@
...
+    "gulp-sourcemaps": "3.0.0",
+    "gulp-svgstore": "9.0.0",
+    "postcss": "8.4.8",
+    "postcss-csso": "5.0.1",
+    "sass-embedded": "1.50.1"
```
Comment (thisisdano) [inline package.json:38]: Update `sass-embedded`

File: package.json [hunk 8]
```diff
@@ -24,17 +24,21 @@
...
+  },
+  "devDependencies": {
+    "@uswds/uswds": "^3.0.0",
+    "uswds": "^2.13.3"
```
Comment (thisisdano) [inline package.json:42]: Using USWDS 2.x and 3.x here are testing dependencies

**Comment by bkline** [comment]
Well done. Thank you! 👏 🙌

**Pull request merged**